packages/cli/src/args.ts

```diff
@@ -5,6 +5,8 @@ import { getPkgName } from './util/pkg-name';
 export const help = () => `
   ${chalk.bold(`${logo} ${getPkgName()}`)} [options] <command | path>
 
+  ${chalk.dim('For deploy command help, run `vercel deploy --help`.')}
```

Comment:
```suggestion
  ${chalk.dim('For deploy command help, run `vercel deploy --help`')}
```

Nit pick: I dont think we end with a period